src/lib/email.ts

```
@@ -0,0 +1,129 @@
+        console.error(err);
+        reject(err);
+        return;
+      }
+
+      // Log the message ID and preview URL if available.
+      console.log("Message sent:", info.messageId);
+      const testUrl = getTestMessageUrl(info);
+      if (testUrl) console.log("Preview URL:", testUrl);
+      resolve(info);
+    });
+  });
```

Biome (1.9.4)

[error] Promise executor functions should not be `async`.



(lint/suspicious/noAsyncPromiseExecutor)